source/classes.tex

```diff
@@ -757,8 +757,9 @@
 \exitexample
 
 \pnum
-Two standard-layout struct (Clause~\ref{class}) types are layout-compatible if
-their common initial sequence comprises all members and bit-fields of
+\indextext{layout-compatible!classes}%
+Two standard-layout struct (Clause~\ref{class}) types are \defn{layout-compatible classes}
```

Comment:
I think the term we put in the index here should be "layout-compatible class" (or maybe "layout-compatible!class") "layout-compatible classes". Use \defnx to specify different index text from the body text.